Source/WebCore/style/StyleBuilderCustom.h

```diff
@@ -2034,7 +2036,31 @@ inline void BuilderCustom::applyValueColor(BuilderState& builderState, CSSValue&
         auto color = builderState.colorFromPrimitiveValue(primitiveValue, ForVisitedLink::Yes);
         builderState.style().setVisitedLinkColor(resolveColor(color));
     }
+
+    builderState.style().setDisallowsFastPathInheritance();
```

Comment:
Why is this being added here?

Reply:
Oh wait, I think this is just Git showing a weird diff, this is really being added for `applyInheritColor`...

Reply:
lol ya the diff is quite strangely formatted